frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -134,7 +134,7 @@ export default {
   methods: {
     async addSection () {
       this.isAdding = true
-      await this.api.post('/content-type/storyboards', {
+      await this.api.post(this.contentNode.sections(), {
         contentNodeId: this.contentNode.id,
```

Comment:
Nach mergen von PR #1274 könnte man sich sogar die Zeile
`contentNodeId: this.contentNode.id`
sparen

Reply:
Aha! Gefällt mir :)